src/main/scala/io/iohk/ethereum/blockchain/sync/RegularSyncController.scala

```diff
@@ -10,108 +10,96 @@ import io.iohk.ethereum.network.p2p.messages.PV62._
 import io.iohk.ethereum.utils.Config
 import org.spongycastle.util.encoders.Hex
 
+import scala.concurrent.duration._
 import scala.concurrent.ExecutionContext.Implicits.global
-import scala.util.Try
 
 trait RegularSyncController {
   self: SyncController =>
 
   private var headersQueue: Seq[BlockHeader] = Seq.empty
-  private var resolvingBranch = false
   private var broadcasting = false
-  private var timeout: Option[Cancellable] = None
 
   import Config.FastSync._
 
   def regularSync(): Receive = {
 
     case StartSyncing =>
+      scheduler.schedule(0.seconds, printStatusInterval, context.self, PrintStatus)
```

Comment:
actually we don't cancel this schedule in `FastSync`. In case of fast -> regular switch we'll get the messages twice